src/Cohesity.Powershell/Scripts/View/Copy-CohesityView.ps1

```diff
@@ -0,0 +1,120 @@
+function Copy-CohesityView {
+    <#
+        .SYNOPSIS
+        Clones the specified Cohesity View.
+        .DESCRIPTION
+        Clones the specified Cohesity View.
+        .NOTES
+        Published by Cohesity
+        .LINK
+        https://cohesity.github.io/cohesity-powershell-module/#/README
+        .EXAMPLE
+        Copy-CohesityView -TaskName "Task-clone-a-view" -SourceViewName "source-view" -TargetViewName "target-view" -TargetViewDescription "Create a view clone" -QosPolicy "Backup Target Low" -JobId 12345
+    #>
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$TaskName,
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$SourceViewName,
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$TargetViewName,
+        [Parameter(Mandatory = $true)]
+        [ValidateNotNullOrEmpty()]
+        [string]$TargetViewDescription,
```

Comment:
Why is description mandatory ? can we make this optional please ?

Reply:
Fixed